app/models/sample.rb

```diff
@@ -747,7 +748,14 @@ def kickoff_pipeline
 
     pr.alignment_config = AlignmentConfig.find_by(name: alignment_config_name) if alignment_config_name
     pr.alignment_config ||= AlignmentConfig.find_by(name: AlignmentConfig::DEFAULT_NAME)
-    pr.save
+    pr.save!
+  rescue StandardError => err
+    LogUtil.log_err_and_airbrake("Error saving pipeline run: #{err.inspect}")
+    # This may cause a message to be shown to the user on the sample page.
+    # See app/assets/src/components/utils/sample.js
+    # Use low-level update_columns to avoid callbacks, because kickoff_pipeline
+    # may be running in a callback already.
+    update_columns(upload_error: Sample::UPLOAD_ERROR_PIPELINE_KICKOFF) # rubocop:disable SkipsModelValidations
```

Comment:
Can you add some sort of "do not try this at home" kind of message? Do not want this to be used as a precedent.